.github/workflows/cyberduck-release-windows.yml

```diff
@@ -0,0 +1,161 @@
+    - name: run batch command
+      shell: cmd
+      run: FOR %%c in ("%GITHUB_REPOSITORY%\windows\target\release\Cyberduck-*") DO "C:\Program Files\Cyberduck CLI\duck.exe" --upload s3:/release.cyberduck.io/ "%%c" --username %AWS_ACCESS_KEY_DEPLOYMENT% --password %AWS_SECRET_KEY_DEPLOYMENT% --retry --quiet --existing overwrite --region us-east-1
+    - name: Upload Artifacts
+      uses: actions/upload-artifact@v3.1.3
+      if: always()
+      with:
+        path: |-
+          windows\target\release\*.exe
+          windows\target\release\*.msi
+          windows\target\release\*.appx
+          !**/*~
+          !**/#*#
+          !**/.#*
+          !**/%*%
+          !**/._*
+          !**/CVS
+          !**/CVS/**
+          !**/.cvsignore
+          !**/SCCS
+          !**/SCCS/**
+          !**/vssver.scc
+          !**/.svn
+          !**/.svn/**
+          !**/.DS_Store
+          !**/.git
+          !**/.git/**
+          !**/.gitattributes
+          !**/.gitignore
+          !**/.gitmodules
+          !**/.hg
+          !**/.hg/**
+          !**/.hgignore
+          !**/.hgsub
+          !**/.hgsubstate
+          !**/.hgtags
+          !**/.bzr
+          !**/.bzr/**
+          !**/.bzrignore
+#     # Fingerprinter was not converted because the behavior is available by default in GitHub Actions and/or it is not configurable
+#     # Mailer plugin was not converted because GitHub Actions will email the actor after failed build and does not support emailing a list of recipients
+    # Ensure parameter if_key_exists is set correctly
+    - name: Install SSH key
+      uses: shimataro/ssh-key-action@v2.6.1
+      with:
+        key: "${{ secrets.VERSION_CYBERDUCK_IO_SSH_KEY }}"
+        name: id_rsa-version_cyberduck_io
+        known_hosts: "${{ secrets.VERSION_CYBERDUCK_IO_KNOWN_HOSTS }}"
+        if_key_exists: replace # replace will allow us to update the ssh-key on an existing build
+        config: |
+          Host VERSION_CYBERDUCK_IO
+            HostName ${{ secrets.VERSION_CYBERDUCK_IO_HOST_NAME }}
+            User ${{ secrets.VERSION_CYBERDUCK_IO_USER }}
+            IdentityFile ~/.ssh/id_rsa-version_cyberduck_io
+      if: always()
+    - name: setup file transfer file
+      uses: actions/github-script@v7.0.0
+      with:
+        script: |-
+          const fs = require('fs').promises
+          const path = require('path')
+          const patterns = "windows/target/update/changelog.*,!**/*~,!**/#*#,!**/.#*,!**/%*%,!**/._*,!**/CVS,!**/CVS/**,!**/.cvsignore,!**/SCCS,!**/SCCS/**,!**/vssver.scc,!**/.svn,!**/.svn/**,!**/.DS_Store,!**/.git,!**/.git/**,!**/.gitattributes,!**/.gitignore,!**/.gitmodules,!**/.hg,!**/.hg/**,!**/.hgignore,!**/.hgsub,!**/.hgsubstate,!**/.hgtags,!**/.bzr,!**/.bzr/**,!**/.bzrignore"
```

Comment:
Same question here with all the excludes.

Reply:
It would be nice if we can dogfood Cyberduck CLI here. We have a _Cyberduck CLI Github Action_ [^1] to transfer artifacts. Documentation can be found in the README [^2] and sample usage in our connection profiles synchronization with S3 [^3].

[^1]: https://github.com/iterate-ch/cyberduck-cli-action
[^2]: https://github.com/iterate-ch/cyberduck-cli-action/blob/main/README.md
[^3]: https://github.com/iterate-ch/profiles/blob/main/.github/workflows/profiles-sync.yml 

@AliveDevil May assist here when in doubt.